data/txs.go

```diff
@@ -58,22 +58,43 @@ func GetExtraPylons(user User) (string, error) {
 }
 
 func Hunt(user User, item Item, getInitialCoin bool) (string, error) {
+
+	defaultCharacter := user.GetDefaultCharacter()
+	defaultCharacterID := ""
+	if defaultCharacter != nil {
+		defaultCharacterID = defaultCharacter.ID
+	}
 	rcpName := "LOUD's hunt without sword recipe"
+	itemIDs := []string{defaultCharacterID}
 
-	itemIDs := []string{}
 	if getInitialCoin {
 		rcpName = "LOUD's get initial coin recipe"
+		itemIDs = []string{}
 	}
 
 	switch item.Name {
 	case WOODEN_SWORD, COPPER_SWORD:
 		rcpName = "LOUD's hunt with a sword recipe"
-		itemIDs = []string{item.ID}
+		itemIDs = []string{defaultCharacterID, item.ID}
 	}
 
 	return ExecuteRecipe(user, rcpName, itemIDs)
 }
 
+func BuyCharacter(user User, item Item) (string, error) {
+	rcpName := ""
+	switch item.Name {
```

Comment:
lets extract this whole `switch` case into a function called `getRecipeNameForItemName`

Reply:
getRecipeNameForItemName does not make sense for me, this function is not that long and I think it would be okay to keep it as it is since too many functions can make code messy.